### DISCUSSION TITLE
Can I add 3:2 resolutuons?

### DISCUSSION THREAD

**Discussion by dpwcast123** [VirtualDrivers/Virtual-Display-Driver#119] [Q&A] [answered]
I'm using a new surface laptop to remote into a pc.   Can 3:2 resolutions be added?   2736x1824   2160x1440  2560x1700  etc.

**Comment by james-s-w-clark** [comment, marked as answer]
You can edit options.txt, reinstall driver, and let us know if it works :)

**Reply by CoolCat64** [reply]
I added
2736, 1824, 60
to the list in option.txt and reinstalled as suggested...this worked for me!
